src/script/view_model/CallingViewModel.ts

```diff
@@ -139,9 +139,16 @@ export class CallingViewModel {
     };
 
     const startCall = async (conversation: Conversation, callType: CALL_TYPE): Promise<void> => {
-      const keyData = conversation.isUsingMLSProtocol
+      const subconversationData = conversation.isUsingMLSProtocol
         ? await core.service!.mls.joinConferenceSubconversation(conversation)
         : undefined;
+
+      const keyData = subconversationData && {
+        epoch: subconversationData.subconversation.epoch,
+        keyLength: subconversationData.keyLength,
+        secretKey: subconversationData.secretKey,
+      };
```

Comment:
🤔  are you sure about this change. The `joinConferenceSubconversation` should return this exact same payload, doesn't it? 
![image](https://user-images.githubusercontent.com/1090716/210793768-3c5e761e-7813-4dbd-ab24-98fe9bf14149.png)

Reply:
I've updated it on my branch: https://github.com/wireapp/wire-web-packages/pull/4748/files